Eatery/Controllers/BRB/AboutTableViewController.swift

```diff
@@ -13,6 +13,7 @@ import SafariServices
 protocol AboutTableViewControllerDelegate: AnyObject {
 
     func aboutTableViewControllerDidLogoutUser(_ stvc: AboutTableViewController)
+    func didClickBackButton()
```

Comment:
Can we rename this to be more stylistically like other delegates? Include the caller (`AboutTableViewController`) and also rename `click` to `tap` for iOS.